### DISCUSSION TITLE
24/11/18 WAVE 2(배포, 모니터링) KPT 회고

### DISCUSSION THREAD

**Discussion by laigasus** [cloud2scape/.github#41] [Announcements]
### Keep
좋았던 점, 앞으로도 계속되었으면 하는 점

### Problem
문제점, 아쉬웠던 점

### Try
앞으로의 일정 계획 등 keep, problem 종합해서 기술

### 이번주

**Comment by laigasus** [comment]
### Keep
- 팀 관계

### Problem
- 피로사회에 취약한 나

### Try
- 앞으로 사고치지 않기

### 이번 주
- [x] Replication DB 연결(+CQRS 패턴)
- [x] K8s Probe 해결(spring actuator)
- [x] 개발환경 내 Kafka 구축
- [x] 비즈니스 로직 내 Saga 패턴 적용(구매 실패 시나리오)
- [x] K8s 롤링 업데이트 적용
- [x] HPA
- [x] 배치 스케줄러용 코드 작성(일괄 삭제 쿼리)
- [x] 품질 테스트(SW) 초안 작성
- [x] 시스템 검증 방법에 대한 자료 수집

**Comment by gimsoomin** [comment]
### Keep
* 납기

### Problem
* 열정

### Try
- [ ] 그만하고 싶다 rds

### 이번주
- [ ] 성능분석 책보기

**Comment by ssedogdoom** [comment]
### Keep
- 이성

### Problem
- 없음

### Try
- 인프라에 대한 더 좋은 효율성 및 여러 방면 알아보기 

### 이번 주
- [x] 인프라 띄우기
- [x] S3 구축 및 연결(GLUE 고려 중) 
- [x] 장애 시나리오 생각 및 인프라 품질 높이기
- [x] MSA 아키대로 인프라 전체 연결 성공 제발
- [x] 테라폼 코드 리팩토링 이나 ArgoCD 구축 및 연결

**Comment by pcj1792** [comment]
### Keep
- Mental health

### Problem
- loki 참 어렵다...

### Try
- 그라파나 커스텀

### 이번 주
- [x] 데이터 가공
- [ ] grafana 로그 안정화
- [ ] aws monitoring
